ReactiveCocoa/Swift/Event.swift

```diff
@@ -6,7 +6,15 @@
 //  Copyright (c) 2015 GitHub. All rights reserved.
 //
 
-import LlamaKit
+import Result
+
+public final class Box<T> {
```

Comment:
Is there a reason to not keep this `internal`?

I would also move it to its own file.